owasp-top10-2017-apps/a7/gossip-world/app/model/db.py

```diff
@@ -2,202 +2,229 @@
 # -*- coding: utf-8 -*-
 
 import MySQLdb
+
+
 class DataBase:
-    def __init__ (self, host, user, password, database):
+    def __init__(self, host, user, password, database):
         self.host = host
         self.user = user
         self.password = password
         self.database = database
-        self.db = MySQLdb.connect(host=self.host,user=self.user,passwd=self.password,db=self.database,charset='utf8')
```

Comment:
Did you miss this following line or you consider it as an unnecessary one? 😃
```python
        self.db = MySQLdb.connect(host=self.host,
                                  user=self.user,
                                  passwd=self.password,
                                  db=self.database,
                                  charset='utf8')
```

Reply:
I've changed it to a call to `self.connect()` which does the same thing.

We could refactor it further, ensuring that the creator of `DataBase` calls `db.connect()`, this way we are not trying to connect on the constructor method of the class.

Reply:
That's awesome! Thanks for this info. 😃